Documentation/check-build.sh

```diff
@@ -85,4 +85,11 @@ fi
 # fi
 
 echo "Building documentation (${target})..."
-exec sphinx-build -M "${target}" "${script_dir}" "${build_dir}" $@
+sphinx-build -M "${target}" "${script_dir}" "${build_dir}" $@ -q 2> >(tee "${warnings}" >&2)
```

Comment:
I'm curious, what's the difference between having the `exec` at the start vs. not?